addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -483,8 +483,8 @@ QUnit.test('display channel mention suggestions on typing "#"', async function (
         public: "groups",
     });
     await this.start();
-    const composer = this.env.models['mail.composer'].create();
-    await this.createComposerComponent(composer);
+    const thread = this.env.models['mail.thread'].create({ id: 100, model: 'res.partner' });
+    await this.createComposerComponent(thread.composer);
```

Comment:
What did you change since the last time, and why is there still only this specific test changed and not the others in this file?

Reply:
Can we revert this change? And if not why?

Reply:
Obviously, I wrong about this change since I don't remember why I did it in the first place and that the run bot is green without it.